src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -188,11 +190,21 @@ public JsonApiException toApiException(String format, Object... args) {
     return new JsonApiException(this, message + ": " + String.format(format, args));
   }
 
+  public JsonApiException toApiException(
```

Comment:
Factory method for passing HTTP status code along: needs to be the first argument due to varargs needed to pass format String arguments.